edk2toolext/invocables/edk2_ci_build.py

```diff
@@ -252,8 +253,17 @@ def Go(self):
                     # perhaps we should ask the validator to run on the package for this target
 
                     # Still need to see if the package decided this should be skipped
-                    if pkg_plugin_configuration is None or\
-                            "skip" in pkg_plugin_configuration and pkg_plugin_configuration["skip"]:
+                    should_skip = False
+                    if pkg_plugin_configuration is None:
+                        should_skip = True
```

Comment:
By not supplying a per-package configuration, the package is skipping all plugins?